### DISCUSSION TITLE
如果要把www的Domain name指向Web3URL制作的网站，有什么比较好的方法？

### DISCUSSION THREAD

**Discussion by punkcanyang** [IntensiveCoLearning/Web3-URL#160] [Q&A] [answered]
最好是网址显示也能保持原本的不变

**Comment by qzhodl** [comment, marked as answer]
ethstorage.io 和 web3url.io 都是采用web3url制作的网站，且使用domain name的cname指向了web3url。具体可以参考这个：https://mxtoolbox.com/SuperTool.aspx?action=cname%3aweb3url.io&run=toolpage
